src/PeselValidator/PeselValidator.cs

```diff
@@ -9,6 +9,8 @@ public static class PeselValidator
 
         public static bool IsValid(string pesel)
         {
+            var a = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'a' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)